packages/meditrak-server/src/tests/dhis/EventBuilder.test.js

```diff
@@ -38,9 +43,20 @@ describe('EventBuilder', () => {
     enrollmentSpy = sinon.stub(Enrollments, 'enrollTrackedEntityInProgramIfNotEnrolled');
     await buildAndInsertSurveys(models, Object.values(SURVEYS));
     const entities = Object.values(ENTITIES);
+    const exploreHierarchy = await findOrCreateDummyRecord(models.entityHierarchy, {
+      name: 'explore',
+    });
     for (let i = 0; i < entities.length; i++) {
       // Upsert entities in order for correct parent/child relationships
-      await upsertEntity(entities[i]);
+      const entity = await upsertEntity(entities[i]);
+      if (entity.parent_id) {
+        await upsertDummyRecord(models.ancestorDescendantRelation, {
+          ancestor_id: entity.parent_id,
+          descendant_id: entity.id,
+          entity_hierarchy_id: exploreHierarchy.id,
+          generational_distance: 1,
+        });
```

Comment:
If we find that we need this kind of functionality often (it's also used in `Entity.test.js`), we may want to abstract it behind a util that elegantly creates both the entities and the hierarchies at the same time

An example

```js
const entities = []; // flat array of entities
const hierarchy = {
  TO: {
    TO_Tefisihc: {
      // ...etc
    },
    TO_Taaneamch: {},
  },
};

const buildEntitiesAndHierarchy = ({ entities, hierarchy }) => {};
```

No need to do this in this PR, or do it at all